docs/modules/publisher-directmethods.md

```diff
@@ -65,46 +61,48 @@ The `_V1` direct methods  uses the  payload schema as described below:
+| `OpcPublishingIntervalTimespan` | No        | String  | null    | The publishing interval for the monitored item to be <br>published. Value expressed in Timespan <br>string({d.hh:mm:dd.fff}). <br>This value will be overwritten when a publishing interval <br>is explicitly defined in the DataSetWriter for the specified node.<br>The value is used as defined in the OPC UA specification. |
+| `DataSetFieldId`                | No        | String  | null    | A user defined tag used to identify the Field in the <br>DataSet telemetry message when publisher runs in <br>PubSub message mode.                                                                                                                                                                                |
+| `DisplayName`                   | No        | String  | null    | A user defined tag to be added to the telemetry message <br>when publisher runs in Samples message mode.                                                                                                                                                                                                          |
+| `HeartbeatInterval`             | No        | Integer | 0       | The interval used for the node to publish a value (a publisher <br>cached one) even if the value hasn't been <br>changed at the source. This value is represented in seconds. <br>0 means the heartbeat mechanism is disabled. <br>This value is ignored when HeartbeatIntervalTimespan is present               |
+| `HeartbeatIntervalTimespan`     | No        | String  | null    | The interval used for the node to publish a value (a publisher <br>cached one) even if the value hasn't been <br>changed at the source. This value is represented in seconds. <br>Value expressed in Timespan string({d.hh:mm:dd.fff}).                                                                          |
+| `SkipFirst`                     | No        | Boolean | false   | Instructs the publisher not to add to telemetry the<br> Initial DataChange (after subscription activation) for this OpcNode.                                                                                                                                                                                      |
+| `QueueSize`                     | No        | Integer | 1       | The desired QueueSize for the monitored item to be published.                                                                                                                                                                                                                                                     |
 
 _Note_: __Id__ field may be omitted when ExpandedNodeIdId is present.
 
```

Comment:
This is confusing for me: Id is Mandatory, how can it then be ommited? The explanation must be improved and the "Mandatory" flag aligned with the explanation.

Reply:
Moved the note to the description parts of `Id` and `ExpandedNodeId` to be more explicit.